Assets/_BForBoss/_Core/Scripts/Managers/WaveManager.cs

```diff
@@ -36,7 +37,7 @@ private void OnEnemySpawned()
         {
             if (_waveModel.IsMaxEnemySpawnedReached)
             {
-                Debug.Log("<color=Blue>Max Enemy Reached</color>");
+                Logger.LogString("<color=Blue>Max Enemy Reached</color>");
```

Comment:
How do you differentiate between the different logs in different places? 
There are logs in WaveModel as well, which i would group together with this.

Reply:
Ah I missed WaveModel, should have updated it now though. Scripts are grouped by namespace, so Waves will all be under the root BForBoss toggle at the moment

Reply:
Is there a way to group it by a Key that we pass in? instead of namespaces?
`Logger.LogString("<color=Blue>Max Enemy Reached</color>", "Key");`

Reply:
I guess if we hard code in preset keys? I was initially thinking we could do something like use reflection to find all the keys used across the project and build the editor that way, but it seems to be impossible without using like roslyn analyzers.

Reply:
It would be a lot messier than the current approach, but having a singleton scriptable object that acted as a key repository could work